ds_judgements_public_ui/templates/includes/results_filters_inputs.html

```diff
@@ -10,7 +10,7 @@
 </div>
 <div class="structured-search__multi-fields-panel">
   <div class="structured-search__limit-to-container">
-    <fieldset>
+    {% if context.errors %}<fieldset class="set-1">{% endif %}
```

Comment:
A tiny stylistic thing, but i think i'd prefer the condition to be *inside* the class here, to make it clear what's actually varying and cut down on duplication, so for instance:

```
<fieldset class="{% if context.errors %}set-1{% endif %}">
```

(although remember that the classname will be different, see my comment above)

Reply:
oh so _this_ is how you do it!

we were doing `<fieldset {% if context.errors %}class="set-1"{% endif %}">`

Reply:
@timcowlishaw @anthonyhashemi updated to`<fieldset class="{% if context.errors %}with-errors{% endif %}">`